deployment/migration-assistant-solution/lib/solutions-stack-eks.ts

```diff
@@ -0,0 +1,236 @@
+import {
+    Aws,
+    CfnMapping, CfnOutput,
+    CfnParameter,
+    Fn,
+    Stack,
+    StackProps,
+    Tags
+} from 'aws-cdk-lib';
+import {Construct} from 'constructs';
+import {
+    GatewayVpcEndpoint,
+    GatewayVpcEndpointAwsService,
+    IVpc,
+    InterfaceVpcEndpoint,
+    InterfaceVpcEndpointAwsService,
+    IpAddresses,
+    IpProtocol,
+    Vpc
+} from "aws-cdk-lib/aws-ec2";
+import {Application, AttributeGroup} from "@aws-cdk/aws-servicecatalogappregistry-alpha";
+import {EKSInfra} from "./eks-infra";
+
+export interface SolutionsInfrastructureStackEKSProps extends StackProps {
+    readonly solutionId: string;
+    readonly solutionName: string;
+    readonly solutionVersion: string;
+    readonly codeBucket: string;
+    readonly createVPC: boolean;
+    readonly stackNameSuffix?: string;
+}
+
+interface ParameterLabel {
+    default: string;
+}
+
+function applyAppRegistry(stack: Stack, stage: string, infraProps: SolutionsInfrastructureStackEKSProps): string {
```

Comment:
Can we move the shared functionality into a separate file?  This is already duplicated in the other stack.

Reply:
Added shared functionality to separate class